modules/microshift-restoring-backups.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift-auto-recover-manual-backup.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-restoring-backups_{context}"]
+= Restoring backups
+
+Backups can be restored after updates or after other system events that remove or damage required data.
+
+Use the following procedure to restore backups.
+
+.Prerequisites
+
+
+
+.Procedure
+
+. Restore backup from the parent directory by running the following command:
```

Comment:
Same thing as in comment regarding backup: I'd drop the notion of "parent directory" as it's relative and might be confusing.

Reply:
done